c_common/front_end_common_lib/src/recording.c

```diff
@@ -538,7 +544,8 @@ static void recording_dma_finished(uint unused, uint tag) {
 }
 
 bool recording_initialize(
-        address_t recording_data_address, uint32_t *recording_flags) {
+        address_t recording_data_address, uint32_t *recording_flags,
+        uint32_t *words_read) {
     struct recording_data_t *recording_data =
```

Comment:
Not really a comment on this change per se - I think it's fine - but there's some calls of this function elsewhere (e.g. spinnaker_graph_front_end/examples/hello_world/src/hello_world.c line 96, and probably other GFE examples too) that need to be edited to match this change.

Reply:
buger. just noticed i dont even have  GFE repo in this set of the tools...... bugger. will solve once the time stuff is done

Reply:
fixed. new pr and added to all pr links